packages/server/lib/project_utils.ts

```diff
@@ -59,14 +43,20 @@ export const getSpecUrl = ({
   // App routes to spec with convention {browserUrl}#/specs/runner?file={relativeSpecPath}
   if (process.env.LAUNCHPAD) {
```

Comment:
Can we remove `process.env.LAUNCHPAD` now everywhere?

Reply:
Ooooh yes let’s do it

Reply:
yes https://github.com/cypress-io/cypress/pull/19778/files